easybuild/easyconfigs/m/Mesa/Mesa-18.1.1-fosscuda-2018b.eb

```diff
@@ -0,0 +1,74 @@
+# Automatically converted from Mesa-17.2.4-intel-2017b.eb
+# Original message:
+# the purpose of the easyconfig is to build a Mesa for software rendering,
+# not hardware rendering. This means you want at least SSE4.2. We build:
+# - llvmpipe: the high-performance Gallium LLVM driver
+# - swr: Intel's OpenSWR
+# it will try to use the llvmpipe by default. It you want swr, do:
+# GALLIUM_DRIVER=swr
+
+easyblock = 'ConfigureMake'
+
+name = 'Mesa'
+version = '18.1.1'
+
+homepage = 'http://www.mesa3d.org/'
+description = """Mesa is an open-source implementation of the OpenGL specification -
+ a system for rendering interactive 3D graphics."""
+
+toolchain = {'name': 'fosscuda', 'version': '2018b'}
+# swr detects and builds parts specific for AVX and AVX2. If we use
+# -xHost, this always gets overwritten and will fail.
+toolchainopts = {'optarch': False}
+
+source_urls = [
+    'https://mesa.freedesktop.org/archive/',
+    'https://mesa.freedesktop.org/archive/%(version)s',
+    'ftp://ftp.freedesktop.org/pub/mesa/%(version)s',
+    'ftp://ftp.freedesktop.org/pub/mesa/older-versions/%(version_major)s.x/%(version)s',
+]
+sources = [SOURCELOWER_TAR_XZ]
+patches = ['Mesa-17.3.6_fix-strip-llvm-flags.patch']
+checksums = [
+    'd3312a2ede5aac14a47476b208b8e3a401367838330197c4588ab8ad420d7781',  # mesa-18.1.1.tar.xz
+    '5aa4e92ed96e3d47ffbecd1ec3a1642407dff11995c5585eb5e06c396654ee30',  # Mesa-17.3.6_fix-strip-llvm-flags.patch
+]
+
+builddependencies = [
+    ('flex', '2.6.4'),
+    ('Bison', '3.0.5'),
+    ('Autotools', '20180311'),
+    ('pkg-config', '0.29.2'),
+    ('Mako', '1.0.7', '-Python-2.7.15'),
+    ('libxml2', '2.9.8'),
+    ('expat', '2.2.5'),
+]
+
+dependencies = [
+    ('zlib', '1.2.11'),
+    ('nettle', '3.4'),
+    ('libdrm', '2.4.92'),
+    ('LLVM', '6.0.0'),
+    ('X11', '20180604'),
+    ('libunwind', '1.2.1'),
+]
+
+# GLU is not part anymore of Mesa package!
+configopts = " --disable-osmesa --enable-gallium-osmesa --enable-gallium-llvm --enable-glx "
```

Comment:
@akesandgren You dropped `--disable-dri` and `--disable-egl` here, while the exact opposites are being added in #6487 (which I'm hoping to merge soon as well).

Reply:
Yes, because they are required for WebKitGTK+ as noted in the description.

Reply:
So there is no conflict here, removing --disable-dri --disable-egl is the same as using --enable-dri --enable-egl. Enable is the default for dri and egl. So yes I can update it to explicitly say --enable if you prefer that.